src/app/feature/feedback/feedback.service.ts

```diff
@@ -44,8 +44,14 @@ export class FeedbackService {
     private userMetaService: UserMetaService,
     private toastController: ToastController,
     private dbService: DbService,
-    private dbSyncService: DBSyncService
+    private dbSyncService: DBSyncService,
+    private appConfigService: AppConfigService
   ) {
+    this.appConfigService.appConfig$.subscribe((appConfig: IAppConfig) => {
```

Comment:
question
Just to check in with your understanding. Some services which use constructor methods will be initialised on import, and potentially before other services have had their init methods called. Could this cause problems if the appConfigService sets the initial config during init? Will we see a flash of old config/layout before new one, and potentially not applying the correct defaults to a service like this?